protoc_plugin/lib/names.dart

```diff
@@ -506,6 +506,23 @@ final List<String> forbiddenExtensionNames = <String>[
   ..._generatedMessageNames
 ];
 
+final List<String> serviceReservedMemberNames = <String>[
+  ..._dartReservedWords,
+  ..._serviceNames,
+];
+
+final List<String> _serviceNames = <String>[
```

Comment:
Can this be `const`? Also, do we need the explicit type on the left-hand side?

Reply:
yes!

Done - also in surrounding code.